ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/reloperators/HiveTableScan.java

```diff
@@ -137,13 +138,15 @@ public HiveTableScan copyIncludingTable(RelDataType newRowtype) {
   }
 
   @Override public RelWriter explainTerms(RelWriter pw) {
-    if (this.useQBIdInDigest) {
-      // TODO: Only the qualified name should be left here
-      return super.explainTerms(pw)
-          .item("qbid:alias", concatQbIDAlias);
-    } else {
-      return super.explainTerms(pw).item("table:alias", tblAlias);
-    }
+    return super.explainTerms(pw)
```

Comment:
Check inheritances of explainTerms & computeDigest

Reply:
`computeDigest` can be safely removed from `HiveTableScan` as it has been removed from `AbstractRelNode`.